module/VuFind/src/VuFind/RecordTab/Formats.php

```diff
@@ -50,9 +50,9 @@ class Formats extends AbstractBase
      * Constructor
      *
      * @param bool $enabled is this tab enabled?
-     * @param bool $urc     use recaptcha?
+     * @param bool $uc     use captcha?
```

Comment:
This looks like a dead parameter -- maybe a copy and paste error. If it's truly unused, maybe you can open a separate PR to simply delete it, and then it's one less detail to worry about here.

Reply:
OK! see #1580

Reply:
Thanks! I've merged that, so if you merge master back into this branch, these diffs should go away. Let me know when you're ready for me to review the rest again!